CHANGES.md

```diff
@@ -1,5 +1,44 @@
 # Changelog
 
+# 1.7.8 (core) / 0.23.8 (libraries)
+
+### New
+
+- Backfills created via GQL can have a custom title and description.
+- `Definitions` now has a `get_all_asset_specs` method, which allows iterating over properties of the defined assets
+- [ui] In filter dropdowns, it’s now possible to submit before all the suggestions have been loaded (thanks bmalehorn!)
```

Comment:
I'm gonna be annoying and request that we standardize the formatting for the "thanks" here -- ideally they'd all contain links to the github profile, but they should at least all be the same format